test-cases/longevity/longevity-alternator-10m-scan-12h-ttl-no-lwt-2h-grace.yaml

```diff
@@ -0,0 +1,56 @@
+test_duration: 3000
+# Run ycsb load using the max integer value of 2147483647 (2^31-1) for number of records per client.
+# ( The prepare_write_cmd is just a dummy in order to trigger post_prepare_cql_cmds )
+prepare_write_cmd:
+  - >-
+    bin/ycsb load dynamodb -P workloads/workloadc -threads 1 -p recordcount=11
+    -p fieldcount=2 -p fieldlength=16
+    -p insertstart=0 -p insertcount=10  -p table=usertable_no_lwt
+stress_cmd:
+  - >-
+    bin/ycsb load dynamodb -P workloads/workloadc -threads 20 -p recordcount=4304304301
+    -p fieldcount=2 -p fieldlength=16
+    -p insertstart=0 -p insertcount=2147483600  -p table=usertable_no_lwt -p dynamodb.ttlKey=ttl -p dynamodb.ttlDuration=43200
+
+  - >-
+    bin/ycsb load dynamodb -P workloads/workloadc -threads 20 -p recordcount=4304304301
+    -p fieldcount=2 -p fieldlength=16
+    -p insertstart=2147483600 -p insertcount=2147483600 -p table=usertable_no_lwt -p dynamodb.ttlKey=ttl -p dynamodb.ttlDuration=43200
+
+
+round_robin: true
+
+dynamodb_primarykey_type: HASH_AND_RANGE
+
+n_loaders: 2
+instance_type_db: 'i3.4xlarge'
+n_monitor_nodes: 1
+n_db_nodes: 4
+
+nemesis_class_name: 'NoOpMonkey'
+nemesis_seed: '007'
+nemesis_interval: 30
+nemesis_during_prepare: false
+
+user_prefix: 'alternator-12h-ttl-10m-scan-no-lwt'
+
+alternator_port: 8080
+alternator_use_dns_routing: true
+
+alternator_enforce_authorization: true
+alternator_access_key_id: 'alternator'
+alternator_secret_access_key: 'password'
+
+# Set Alternator TTL parameters
+# ---------------------------------------------
+# TTL mode is experimental in version 5.1.
+# Enable TTL feature in Scylla.
+# Set 'alternator_ttl_period_in_seconds' to 10 minutes for the TTL scan interval.
+experimental: true
```

Comment:
It is not needed for 2022.2. probably should not be needed for 5.1 as well, but this was the instructions got on testing start.
It may be also needed for running 2022.1.

Reply:
if it's not needed please remove, anyhow this feature isn't going to 2022.1, as far as I know. 
and if it would be needed there, we'll add it back.